features/support/env.rb

```diff
@@ -17,33 +17,22 @@
 Capybara.app_host = ENV["GOVUK_WEBSITE_ROOT"]
 phantomjs_logger = File.open("log/phantomjs.log", "a")
 
-BLACKLISTED_URLS = ['www.google-analytics.com']
+GOOGLE_ANALYTICS_URL = 'www.google-analytics.com'
+BLACKLISTED_URLS = [GOOGLE_ANALYTICS_URL]
 
 Capybara.register_driver :poltergeist do |app|
-  Capybara::Poltergeist::Driver.new(app, phantomjs_logger: phantomjs_logger)
+  options = {
+    debug: ENV['POLTERGEIST_DEBUG'] || false,
+    phantomjs_logger: phantomjs_logger,
+    url_blacklist: BLACKLISTED_URLS
```

Comment:
Is this what we used to do here:
https://github.com/alphagov/smokey/pull/260/files
https://github.com/alphagov/smokey/pull/260/files#diff-57013262a6cdf1f0f6cb2c64aa762c68L26

Which caused this: https://github.com/alphagov/smokey/issues/256